### ISSUE TITLE
REPL: Wrong dependency for Ammonite

### ISSUE THREAD

**Issue by Iltotore** [com-lihaoyi/mill#1359]
When using the `module.repl` task, Mill try to download a deleted version of Ammonite:
```scala
com.lihaoyi:ammonite_3.0.0:2.3.8-65-0f0d597f
        not found: C:\Users\rapha\.ivy2\local\com.lihaoyi\ammonite_3.0.0\2.3.8-65-0f0d597f\ivys\ivy.xml
        not found: https://repo1.maven.org/maven2/com/lihaoyi/ammonite_3.0.0/2.3.8-65-0f0d597f/ammonite_3.0.0-2.3.8-65-0f0d597f.pom
```

The latest version is now: [2.3.8-124-2da846d2](https://search.maven.org/artifact/com.lihaoyi/ammonite_3.0.0/2.3.8-124-2da846d2/jar)

**Comment by lefou** [comment]
Ammonite 2.3.8-65-0f0d597f was never released for Scala 3.0.0 and 2.13.6. Unfortunatelly any version bump to ammonite fail the build of mill. See #1337 and #1343. Any help is appreciated.

**Comment by lefou** [comment]
Fixed by #1371

**Comment by dev590t** [comment]
@Iltotore you can also choose a version compatible with your current scala version in https://mvnrepository.com/artifact/com.lihaoyi/ammonite

change the version ammonite to use :
```
// in your build.sc
override def ammoniteVersion: T[String] = T("2.4.0")
```
